### DISCUSSION TITLE
[Q&A] 你们部署最新版本的0.118.1的时候没有出现bug吗？

### DISCUSSION THREAD

**Discussion by 7tinghua** [lobehub/lobehub#883] [Development Question] [answered]
聊天页面不能访问了。会报错。Application error: a client-side exception has occurred (see the browser console for more information).
设置页面和market页面都可以正常访问。但是进入不了聊天页面。
不知道你们是不是也存在这个情况。

**Comment by arvinxx** [comment, marked as answer]
更新 118.2 应该就好了

**Reply by 7tinghua** [reply]
好的 感谢！
